tests/api/search/entities.test.js

```diff
@@ -142,6 +141,60 @@ describe('search:entities', () => {
         foundIds.should.not.containEql(humanB._id)
       })
     })
+
+    describe('claim', async () => {
+      let workAuthor, workWithAuthor
+      before(async () => {
+        workAuthor = await createHuman()
+        workWithAuthor = await createWorkWithAuthor(workAuthor)
+        await waitForIndexation('entities', workWithAuthor._id)
+      })
+
+      it('should reject unknown properties', async () => {
+        await search({ types: 'works', claim: 'wdt:P6=wd:Q535' })
+        .then(shouldNotBeCalled)
+        .catch(err => {
+          err.statusCode.should.equal(400)
+          err.body.status_verbose.should.equal('unknown property')
+        })
+      })
+
+      it('should reject invalid property values', async () => {
+        await search({ types: 'works', claim: 'wdt:P123=456' })
+        .then(shouldNotBeCalled)
+        .catch(err => {
+          err.statusCode.should.equal(400)
+          err.body.status_verbose.should.equal('invalid property value')
+        })
+      })
+
+      it('should find an entity by one of its relation claims', async () => {
+        const results = await search({ types: 'works', claim: `wdt:P50=${workAuthor.uri}`, lang: 'en', filter: 'inv' })
+        const foundIds = _.map(results, 'id')
+        foundIds.should.containEql(workWithAuthor._id)
+      })
+
+      it('should accept OR conditions', async () => {
+        const results = await search({ types: 'works', claim: `wdt:P50=wd:Q535|wdt:P50=${workAuthor.uri}`, lang: 'en', filter: 'inv' })
```

Comment:
now that development is over, what about (independent possibilities): 
- moving describe claim block to its own file
- replace `wd:Q535` by a dynamic value 
- have an inv uri instead of relying on external uri